ElectronClient/gui/PromptDialog.jsx

```diff
@@ -223,7 +234,7 @@ class PromptDialog extends React.Component {
 		if (this.props.inputType === 'datetime') {
 			inputComp = <Datetime value={this.state.answer} inputProps={{ style: styles.input }} dateFormat={time.dateFormat()} timeFormat={time.timeFormat()} onChange={momentObject => onDateTimeChange(momentObject)} />;
 		} else if (this.props.inputType === 'tags') {
-			inputComp = <CreatableSelect styles={styles.select} theme={styles.selectTheme} ref={this.answerInput_} value={this.state.answer} placeholder="" components={makeAnimated()} isMulti={true} isClearable={false} backspaceRemovesValue={true} options={this.props.autocomplete} onChange={onSelectChange} onKeyDown={event => onKeyDown(event)} />;
+			inputComp = <CreatableSelect styles={styles.select} theme={styles.selectTheme} ref={this.answerInput_} value={this.state.answer} placeholder="" components={makeAnimated()} isMulti={true} isClearable={false} backspaceRemovesValue={true} defaultOptions = {this.props.autocomplete} loadOptions={this.filterOptions.bind(this)}  onChange={onSelectChange} onKeyDown={event => onKeyDown(event)} />;
```

Comment:
Looking at this, I feel we don't need the async select. The sorted tags are just derived from `this.props.autocomplete` and you can get this data in a synchronous way, or am I missing something?